internal/gcs/gcs.go

```diff
@@ -18,7 +18,11 @@ var style aurora.Aurora
 
 func IsRunningInGCP() bool {
 	resp, err := http.Get("http://metadata.google.internal")
-	resp.Body.Close()
+
+	if resp != nil && resp.Body != nil {
```

Comment:
This is ok but I don't think we need to check if the body is nil, only the response.

Reply:
Yes but according to the documentation it won't be nil.